src/autobridgecalculator.js

```diff
@@ -70,14 +73,17 @@ AutobridgeCalculator.prototype.calculate = function() {
       continue;
     }
 
-    if (legOneTakerGetsFunded.compareTo(legTwoTakerPaysFunded) > 0) {
+    //  using private fields for speed
```

Comment:
I think generally using private fields is not an acceptable solution for optimization. What was causing the slowdown, the from_json in comparedTo?